Frameworks/SPMySQLFramework/Source/SPMySQLConnection Categories/Copying.m

```diff
@@ -52,6 +52,7 @@ - (id)copyWithZone:(NSZone *)zone
 	[copy setPort:port];
 	[copy setUseSocket:useSocket];
 	[copy setSocketPath:socketPath];
+    [copy setAllowDataLocalInfile:allowDataLocalInfile];
```

Comment:
Mix of spaces and tabs. We have no strict coding convention and this mix is already present in another files, so that's must be not so big trouble :) I just want to know for future if there are some basic rules I need to follow.

Reply:
Good question! I just converted this file to all spaces for consistency but I'll ask the group what we should go with in general!